src/digital/v1_compat.rs

```diff
@@ -1,7 +1,39 @@
-//! v1 compatibility wrapper
-//! this module adds reverse support for v2 digital traits
-//! v2 traits must be explicitly cast to the v1 version using `.into()`,
-//! and will panic on internal errors
+//! v1 compatibility wrappers
+//! 
+//! this module provides wrappers to support use of v2 implementations with
+//! v1 consumers. v2 traits must be explicitly cast to the v1 version using 
+//! `.into()`, and will panic on internal errors
```

Comment:
I don't want to be nitpicky, but correct capitalization and punctuation would be nice.